index.js

```diff
@@ -55,6 +57,20 @@ function addFilesRecursively(builder, baseURL, dir) {
   }
 }
 
+function maybeSignWebBundle(webBundle, opts, logger) {
+  if (!opts.integrityBlockSign || !opts.integrityBlockSign.key) {
+    return webBundle;
```

Comment:
Should we throw an error if `opts.integrityBlockSign` is set but `opts.integrityBlockSign.key` is missing instead of silently not signing the bundle?

Reply:
As discussed offline, I removed the key check, since it's handled by wbn-sign package and will throw an error if trying to parse a key from undefined.